File: HW01/CPP/src/Volleyball.cpp
#include "../include/Volleyball.h"
#include <iostream>
#include <string>
using namespace std;

Volleyball::Volleyball(string material, string size, double price) : IndoorItem(price){
    this->material = material;
    this->size = size;
    this->price = price;
}

void Volleyball::display() {
    cout << "$" << this->price << " - This volleyball is made of " << this->material << " and is " << this->size << endl;
}

File: HW01/PYTHON/project/Volleyball.py
from .IndoorItem import IndoorItem

class Volleyball(IndoorItem):
    def __init__(self, price, material, size):
        super().__init__(price)
        self.material = material
        self.size = size
    
    def display(self):
        print(f"${self.price} - This volleyball is made of {self.material} and is {self.size}", end="")